launcher/game/translations.rpy

```diff
@@ -381,6 +384,21 @@ screen extract_dialogue:
                     textbutton _("Escape quotes and other special characters.") action ToggleField(persistent, "dialogue_escape") style "l_checkbox"
                     textbutton _("Extract all translatable strings, not just dialogue.") action ToggleField(persistent, "dialogue_strings") style "l_checkbox"
 
+                add SPACER
+                add SEPARATOR2
+
+                frame:
+                    style "l_indent"
+                    has vbox
+
+                    text _("Language:")
```

Comment:
```suggestion
                    text _("Language (or None):")
```
This should be simple enough to show creators that they can use None here. If they have put another language and forgotten about the default value, for example.